src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -76,7 +76,7 @@
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.7.6" />
     <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.2.5" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers" Version="0.2.3" />
-    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.Identity" Version="1.2.0-preview.4" />
```

Comment:
We can't ship with a preview version of a dependency. Is this change to 1.2.0 required?

Reply:
No, it's not required. I changed it back to 1.1.1.